web/src/components/Toast.tsx

```diff
@@ -1,6 +1,9 @@
 import * as ToastPrimitive from '@radix-ui/react-toast';
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { FaXmark } from 'react-icons/fa6';
```

Comment:
@tonypls I noticed you used `import { HiOutlineX } from 'react-icons/hi'`, was there some special reason for that?

@Alportan do you have the icon name which you prefer?
(I just want to prevent us from importing duplicate icons/code)

Reply:
Nope, we should try unify our x marks, I think this is our most commonly used one `import { HiXMark } from 'react-icons/hi2';`

Reply:
Great!

I'll switch all the x icons to use that :)